From e03d0cff8b9b1cadebfaefd6cbc110615f47f7e4 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 19 Apr 2023 18:11:26 +0530
Subject: [PATCH] Disable trusty driver

Disabling trusty driver as part of enablement.

Note: This needs to be reverted.

Tracked-On: OAM-108883
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 drivers/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/Makefile b/drivers/Makefile
index 660f7df8d33a..a8485501cde6 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -117,7 +117,6 @@ obj-$(CONFIG_W1)		+= w1/
 obj-y				+= power/
 obj-$(CONFIG_HWMON)		+= hwmon/
 obj-$(CONFIG_THERMAL)		+= thermal/
-obj-$(CONFIG_TRUSTY)		+= trusty/
 obj-$(CONFIG_WATCHDOG)		+= watchdog/
 obj-$(CONFIG_MD)		+= md/
 obj-$(CONFIG_BT)		+= bluetooth/
-- 
2.40.0

